SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,78 @@
+bzz
+===
+
+The bzz protocol consists of a single message type; handshake.
+
+The bzz handshake is the first message to be exchanged between peers
+after the p2p transport protocol has been successfully negotiated. After
+the handshake each peer should remember the following data about the
+other:
+
+-   Swarm Overlay Address of the peer
```

Comment:
Missing definition

Reply:
Sorry, added as separate PR: https://github.com/nolash/SWIPs/pull/4https://github.com/nolash/SWIPs/pull/4 

How do we manage these definitions across md documents? With latex etc this is easy because one can use dictionaries (and bilbiographies to point to external definitions)